Formula/kedge.rb

```diff
@@ -0,0 +1,27 @@
+class Kedge < Formula
+  desc "Simple, Concise & Declarative Kubernetes Applications"
+  homepage "http://kedgeproject.org"
+  url "https://github.com/kedgeproject/kedge/archive/v0.3.0.tar.gz"
+  sha256 "ddb036ff5114f3900b68793baf9c7523b13f67fa85e6d869c0f335bac42e80df"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    mkdir_p buildpath/"src/github.com/kedgeproject"
+    ln_s buildpath, buildpath/"src/github.com/kedgeproject/kedge"
+    system "make", "bin"
+    bin.install "kedge"
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/kedge version")
+
+    # test a simple Kedge file
+    test_file = "name: test\n" \
+                "containers:\n"\
+                "- image: test"
+    (testpath/"kedgefile.yml").write(test_file)
+    system "#{bin}/kedge", "generate", "-f", "kedgefile.yml"
```

Comment:
should `assert_match`, `assert_equal`, or `assert_predicate` something has actually happened here